config/configtest/config_overrides_defaults.yaml

```diff
@@ -0,0 +1 @@
+port: 9000
```

Comment:
How is this any different than `config.yaml`? Is there significance in the port number?

Reply:
Yes. The first one has the exact same port number as the default values so we don't expect a change from the default value. This file on the other hand, carries a port value different than the default value `2424`. I guess it's better to change the first file's name from `config/configtest/config.yaml ` to `config/configtest/config_same_as_defaults.yaml`.